cves/CVE-2015-8213.yml

```diff
@@ -49,7 +49,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A getter function belonging to a utility file in Django existing before the patched version might allow remote attackers to obtain sensitive application secrets via a settings key in place of a date/time format setting. This allowed a malicious user to pass a secret key instead of a date format and obtain said application secret.
```

Comment:
Make sure to limit lines to 80 characters for readability.

Reply:
You switch tense in the middle of the desc here, I'd recommend keeping it all in present progressive